src/WebSocketManager/WebSocketManagerMiddleware.cs

```diff
@@ -23,7 +23,10 @@ public class WebSocketManagerMiddleware
         public async Task Invoke(HttpContext context)
         {
             if (!context.WebSockets.IsWebSocketRequest)
+            {
+                await _next.Invoke(context);
```

Comment:
How does uncommenting the line fix the issue?

From time to time, when the next middleware is invoked (and this is the last middleware) there is an exception thrown.

Did you test this?